pytorch_lightning/trainer/trainer.py

```diff
@@ -230,7 +230,7 @@ def __init__(
 
             num_nodes: number of GPU nodes for distributed training.
 
-            gpus: Which GPUs to train on.
+            gpus: number of gpus to train on (int) or which GPUs to train on (list or str) applied per node
```

Comment:
let's add example of such string

Reply:
not just for the string but for all `int`, `list` and `str` should be added since is an important parameter??

Reply:
We already have a good overview of this in the docs (https://pytorch-lightning.readthedocs.io/en/latest/multi_gpu.html under the 'Select GPU devices') so maybe we should just link to that?

Reply:
yes, lets refer this docs...